letsencrypt/tests/cli_test.py

```diff
@@ -66,12 +66,30 @@ def test_exceptions(self):
         attrs = {'view_config_changes.side_effect' : error}
         self.assertRaises(
             errors.Error, self._call, ['--debug'] + cmd_arg, attrs)
-        self._call(cmd_arg, attrs)
 
         attrs['view_config_changes.side_effect'] = [ValueError]
         self.assertRaises(
             ValueError, self._call, ['--debug'] + cmd_arg, attrs)
-        self._call(cmd_arg, attrs)
+
+    @mock.patch("letsencrypt.cli.sys")
+    def test_handle_exception(self, mock_sys):
+        # pylint: disable=protected-access
+        import StringIO
+        from letsencrypt import cli
+        from letsencrypt import errors
+
+        cli._handle_exception(errors.Error, "detail", None, None)
```

Comment:
`(errors.Error, exc_value="detail", trace=None, args=None)` could make it a bit more readable without having to jump around the files, same below